client/src/components/Navbar.js

```diff
@@ -1,13 +1,24 @@
 import React, {useState} from 'react'
-import Button from "./Button";
+import { Button } from './Button'
 import styles from './Navbar.module.css'
 
 function Navbar() {
   return (
     <>   
         <nav className={styles['Navbar']}>
             <div className={styles['navbar-container']}>
+            <div className={styles['empty-container']}>
+              </div>
```

Comment:
what's the point of this empty container?

Reply:
Yeah all these divs are unnecessary. We only need one `"navbar-container"` and the two `<Button />` inside it. The flex container then should handle the alignment on its own.